src/interactive/documentation.ts

```diff
@@ -2,6 +2,30 @@ import * as path from 'path'
 import * as vscode from 'vscode'
 import { withLanguageClient } from '../extension'
 import { getVersionedParamsAtPosition, setContext } from '../utils'
+import MarkdownIt = require('markdown-it')
+
+const md = new MarkdownIt().
+    use(require('@traptitech/markdown-it-katex'), {
+        output: 'html'
+    }).
+    use(require('markdown-it-highlightjs'), {
+        inline: true, register: {
```

Comment:
Maybe we want to have Julia syntax highlight as default, right ? #1456 does that.